techniques/css/C42.html

```diff
@@ -18,67 +18,50 @@ <h2>When to Use</h2>
 		</section>
 		<section id="description">
 			<h2>Description</h2>
-			<p>The objective of this technique is to ensure that links in navigation or pagination menus will be spaced so that they fall within an area that measures at least 44 x 44 CSS pixels if the target area itself is smaller than that. The aim is to provide an adequate target clearance so the offset to adjacent targets is sufficient to prevent accidental pointer activation of adjacent targets.</p>	
-		
+			<p>The objective of this technique is to ensure that navigation or pagination links will be sized so that they fall within an area that measures at least 44 &#215; 44 CSS pixels.</p>
```

Comment:
sorry for drawing this out, but it's not "they fall within an area that measures...". It's not about the area that the target falls in (i.e. clearance/spacing). it's...the actual size of the target.

As we probably do want to keep an aim, for consistency with other techniques, suggest just grabbing this:

suggest grabbing the very latest suggestion here (as we probably want to keep the aim):

```
<p>The objective of this technique is to ensure that navigation or pagination links measure at least 44 by 44 CSS pixels. The aim is for target sizes to be large enough for users to easily activate them.</p>
```

Reply:
done